Loop/View Controllers/StatusTableViewController.swift

```diff
@@ -450,7 +450,8 @@ final class StatusTableViewController: LoopChartsTableViewController {
             if let predictedGlucoseValues = predictedGlucoseValues {
                 self.statusCharts.setPredictedGlucoseValues(predictedGlucoseValues)
             }
-            if let lastPoint = self.statusCharts.glucose.predictedGlucosePoints.last?.y {
+            if !FeatureFlags.predictedGlucoseChartClampEnabled,
```

Comment:
[nit] This seems like piggybacking hiding the eventual glucose description on clamping the glucose prediction values. At a minimum, I suggest a comment explaining, or consider adding another feature flag to be thoroughly descriptive.

Reply:
I'll add a comment.